applications/application-template.md

```diff
@@ -122,7 +122,7 @@ To assist you in defining it, we have created a document with examples for some
 
 * **Total Estimated Duration:** Duration of the whole project (e.g. 2 months)
 * **Full-Time Equivalent (FTE):**  Average number of full-time employees working on the project throughout its duration (see [Wikipedia](https://en.wikipedia.org/wiki/Full-time_equivalent), e.g. 2 FTE)
-* **Total Costs:** Amount of payment in USD for the whole project. The maximum amount of funding depends on the [level](../README.md#level_slider-level) of your grant application. 
+* **Total Costs:** Requested amount in USD for the whole project. The maximum amount of funding depends on the [level](../README.md#level_slider-levels) of your grant application (e.g. 12,000 USD). This and the costs for each milestone need to be provided in USD; if the grant is paid out in Bitcoin, the amount will be calculated according to the exchange rate at the time of payment.
```

Comment:
Also isn't this really the other way around? I.e. the level of the application depends on the amount of funding requested.

Reply:
```suggestion
* **Total Costs:** Requested amount in USD for the whole project (e.g. 12,000 USD). Note that the acceptance criteria and additional benefits vary depending on the [level](../README.md#level_slider-levels) of funding requested. This and the costs for each milestone need to be provided in USD; if the grant is paid out in Bitcoin, the amount will be calculated according to the exchange rate at the time of payment.
```